### ISSUE TITLE
No longer able to make a get request?

### ISSUE THREAD

**Issue by JoeDawson** [njasm/soundcloud#18]
Hello @njasm,

I have some code that fetched my favourites from SoundCloud - that was working fine a couple of weeks ago.

Here's my current code:

```php
$soundcloud = new Njasm\Soundcloud\Soundcloud(
	// my id,
	// my secret
);

$request	= '/users/51721038/favorites';
$params 	= ['limit' => 10];
$tracks 	= $soundcloud->get($request, $params)->request();

print_r($tracks);
```

I now receive a **400 - Bad Request**, any reason as to why this now giving me this error? Where it was fine before? 

I think it's important for me to note that I did do a `composer update` recently which did pull in some updates but I'm unsure of which they were.

Hope you can help!

**Comment by JoeDawson** [comment]
Was on `dev-master`, jumped back to `2.2.0` - working as normal. Sorry.

**Comment by njasm** [comment]
Thanks for reporting that.. I'm looking into it right now.

**Comment by njasm** [comment]
I found the problem.
I've forgot to add a clause to not send the params in the HTTP body when the method is ``GET``.

I've tested the fix with ``GET``, ``POST``, ``PUT``and ``DELETE`` verbs against the web-profiles of my account. Everything seems to be working as expected, I've also tested your example.

@JoeDawson if you (or anybody) have the change to test this version in a ``DEVELOPMENT`` environment, with the latest ``dev-master``and report here, would be nice, so we can tag it ``2.2.1``

thanks.
